docs/user_guide/Certificates/Official_Certificates.inc

```diff
@@ -1,38 +1,54 @@
 Installing Official Certificates
 --------------------------------
 
-Below are the steps to install official certificates for a SIMP client on
-the SIMP server:
+This section describes how to install infrastructure certificates from an
+official certificate authority on the puppetserver for distribution to client
+servers.  You need to have simp_options::pki set to ``simp`` on the client for
+this to work.
+
+The key distribution directory on the puppetserver is located under the site_files directory.
+The site_files directory is an alternate module path set in each environment's environment.conf file.
+By default the site_files directory is ``/var/simp/environments/simp/site_files/``
+and the key distribution directory is ``pki_files/files/keydist``.  The system expects each client
+to have a directory that is its FQDN and the certs should be named with the FQDN.
+
+To install official certificates on the puppetserver do the following:
 
 1. Copy the certificates received from a proper :term:`CA` to the SIMP server.
-2. Add the keys for the node to ``/var/simp/environments/simp/site_files/pki_files/files/keydist``.
+2. Add the certificates for the node to key disttibution directory in the site_files.
```

Comment:
* 'disttibution' -> 'distribution'
* I preferred the fully-qualified path, as it would be easy to cut and paste.  Was it removed to allow for a different environment?